otcextensions/sdk/cloud/cce.py

```diff
@@ -199,3 +199,216 @@ def delete_cce_cluster(
             self.cce.wait_for_job(obj.job_id, **wait_args)
 
         return None
+
+    def create_cce_cluster_node(
+        self,
+        count=1,
+        root_volume_size=40,
+        root_volume_type='SATA',
+        data_volumes=[{'SATA': 100}],
+        wait=True, wait_timeout=180, wait_interval=5,
+        **kwargs
+    ):
+        """Create CCE cluster node
+
+        :param dict annotations: Annotations.
+        :param str availability_zone: Availability zone of the cluster_node.
+        :param int count: Count of the cluster nodes to be created.
+        :param str cluster: CCE cluster attached to.
+        :param list data_volumes: List of Data volumes attached to the
+            cluster node.
+        :param str dedicated_host: ID of the Dedicated Host to which
+            nodes will be scheduled.
+        :param str ecs_group: ID of the ECS group where the CCE node can
+            belong to.
+        :param str fault_domain: The node is created in the specified fault
+            domain.
+        :param str flavor: Flavor.
```

Comment:
can you specify `FlavorId` or `FlavorName`? (If it matters)

Reply:
done